roles/branch/tasks/main.yml

```diff
@@ -0,0 +1,34 @@
+---
+- name: Remove theme directory
+  file: 
+    path: "{{ theme_dir_main }}/{{ theme_branch }}" 
+    state: absent
+  become_method: sudo
+  become: yes
+  become_user: edxapp
+
+- include: private-repo-ssh.yml
+  when: ("{{ script_mode }}" == 'br')
+
+- name: Clone git theme repo
+  git: 
+    repo: "{{ theme_repo }}"
+    version: "{{ theme_branch }}"
+    dest: "{{ theme_dir_main }}/{{raccoongang_theme_dir_main}}/{{ theme_branch }}"
+    ssh_opts: "-o StrictHostKeyChecking=no"
+    accept_hostkey: yes
```

Comment:
If you have `ssh_opts: "-o StrictHostKeyChecking=no"` option, `accept_hostkey: yes` will be ignored